package.json

```diff
@@ -45,7 +45,7 @@
     "@opentripplanner/geocoder": "^1.4.2",
     "@opentripplanner/humanize-distance": "^1.2.0",
     "@opentripplanner/icons": "^2.0.6",
-    "@opentripplanner/itinerary-body": "^5.1.3",
+    "@opentripplanner/itinerary-body": "^5.1.5",
```

Comment:
Could you use the latest itinerary body package? As of writing, it is now 5.2.1 and includes a Tagalog language fix.